### PR TITLE
[Refactor] Refactor docs about `inference` and `analysis_tools`.

### PR DESCRIPTION
**Pull request by pixeli99**

### CODE REVIEW

File: docs/en/user_guides/3_inference.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+```shell
+python demo/demo_mot_vis.py \
+    ${CONFIG_FILE} \
+    --input ${INPUT} \
+    [--output ${OUTPUT}] \
+    [--checkpoint ${CHECKPOINT_FILE}] \
+    [--score-thr ${SCORE_THR} \
+    [--device ${DEVICE}] \
+    [--backend ${BACKEND}] \
```
Comment (JingweiZhang12) [inline docs/en/user_guides/3_inference.md:54]: backend is not used in visualizatin. Suggest removing this line and remove backend arg in the demo_mot_vis.py

File: docs/en/user_guides/3_inference.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+The `INPUT` and `OUTPUT` support both mp4 video format and the folder format.
+
+Optional arguments:
+
+- `OUTPUT`: Output of the visualized demo. If not specified, the `--show` is obligate to show the video on the fly.
+- `CHECKPOINT_FILE`: The checkpoint is optional in case that you already set up the pretrained models in the config by the key `pretrains`.
+- `SCORE_THR`: The threshold of score to filter bboxes.
+- `DEVICE`: The device for inference. Options are `cpu` or `cuda:0`, etc.
+- `BACKEND`: The backend to visualize the boxes. Options are `cv2` and `plt`.
```
Comment (JingweiZhang12) [inline docs/en/user_guides/3_inference.md:66]: The same as above

File: docs/en/user_guides/3_inference.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+```
+
+The `INPUT` and `OUTPUT` support both mp4 video format and the folder format.
+
+Optional arguments:
+
+- `OUTPUT`: Output of the visualized demo. If not specified, the `--show` is obligate to show the video on the fly.
+- `DEVICE`: The device for inference. Options are `cpu` or `cuda:0`, etc.
+- `--show`: Whether show the video on the fly.
+- `--gt_bbox_file`: The gt_bbox file path of the video. We only use the gt_bbox of the first frame. If not specified, you would draw init bbox of the video manually.
```
Comment (JingweiZhang12) [inline docs/en/user_guides/3_inference.md:113]: - `GT_BBOX_FILE`：

File: docs/en/user_guides/3_inference.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- `DEVICE`: The device for inference. Options are `cpu` or `cuda:0`, etc.
+- `--show`: Whether show the video on the fly.
+
+**Examples:**
+
+Assume that you have already downloaded the checkpoints to the directory `checkpoints/`, your video filename is `demo.mp4`, and your output path is the `./outputs/`
+
+```shell
+python ./demo/demo_vid.py \
+    ./configs/vid/selsa/selsa_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py \
```
Comment (JingweiZhang12) [inline docs/en/user_guides/3_inference.md:35]: Please modify all names of configs to be compatible with dev-1.x/

File: docs/en/user_guides/6_analysis_tools.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+    [--show] \
+    [--backend ${BACKEND}]
+```
+
+The `RESULT_DIR` contains the inference results of all videos and the inference result is a `txt` file.
+
+Optional arguments:
+
+- `OUTPUT`: Output of the visualized demo. If not specified, the `--show` is obligate to show the video on the fly.
+- `--fps`: FPS of the output video.
```
Comment (JingweiZhang12) [inline docs/en/user_guides/6_analysis_tools.md:69]: - `FPS`:

**Review by JingweiZhang12 — APPROVED** [review]

File: docs/en/user_guides/3_inference.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+- `--show`: Whether show the video on the fly.
+
+**Examples of running mot model:**
+
+```shell
+python demo/demo_mot_vis.py \
+    configs/mot/deepsort/sort_faster-rcnn_fpn_4e_mot17-private.py \
+    --input demo/demo.mp4 \
+    --output mot.mp4 \
+```
```
Comment (JingweiZhang12) [inline docs/en/user_guides/3_inference.md:76]: Please explain the difference of  `checkpoint` argument  betwenn deepsort and other MOT trackers.

File: docs/en/user_guides/3_inference.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+- `--show`: Whether show the video on the fly.
+
+**Examples of running mot model:**
+
+```shell
+python demo/demo_mot_vis.py \
+    configs/mot/sort/sort_faster-rcnn_r50_fpn_8xb2-4e_mot17halftrain_test-mot17halfval.py \
+    --input demo/demo.mp4 \
+    --output mot.mp4 \
+```
```
Comment (GT9505) [inline docs/en/user_guides/3_inference.md:76]: 给一个bytetrack需要手动指定ckpt的例子吧

Comment (pixeli99) [reply docs/en/user_guides/3_inference.md:76]: ok

**Review by GT9505 — APPROVED** [review]

**Pull request merged**